### ISSUE TITLE
Misuniderstanding on how patents work

### ISSUE THREAD

**Issue by aquilesC** [FOSH-following-demand/FOSH-following-demand.github.io#25]
On the website, you state:
> ideas are patented and licensed to a company that will then take all steps necessary (and there are a lot of them) to bring these ideas to a commercial state

and:

> But it is bad in the sense that something that had a lot of public investment becomes closed by default.

It makes it look like the process of patenting and licensing is evil, you are stripping the freedom out of publicly funded research. And there are many things to discuss on these two lines: 

1) The owner of the patent is the university/institute/agency that funded the research together with the researcher (percentages vary). Therefore, at the moment of licensing technology, the funding organisms receive money in return. For successful commercial projects, the money received can greatly offset the investment done with public money. One can argue that this cycle shouldn't be the one leading research, etc. but it's beyond the scope. 

2) Patents based on research projects normally are also associated with papers. This makes the idea behind a product publicly accessible (you have to pay the paper access, etc. but I think you get the point).  If researchers would publish thorough methods, any other researcher could build exactly the same device. The only freedom that patents take away from you is the freedom to sell those devices. I see this as a reasonable request. 

3) A clear example of what happens in point 2) would be a microscope. I can buy one from Nikon, Olympus, etc. but I can also build my own. Then you wonder, why so many biologists use commercial microscopes. I have my own ideas, but I believe is a good exercise to do. The same with laser cavities; just open up a Ti:Sapphire, copy the design, buy the crystal and you are done.  

4) It is not illegal to build your own devices and use them for a scientific publication. Therefore, it is not true that research becomes closed by default after a patent or after a company becomes involved. The reality is that the research was closed from day 0, with or without companies involved. Researches don't take the time to publish blueprints, schematics, etc. 

I think that having open-hardware instruments should not be antagonistic of companies and patents, it should be a cultural change on how scientists develop methods and tools to make them accessible. What goes into the value of the product is not just the research, it is its reliability, easy of use, software, design, technical support, etc. And fundamentally, it is time. When you buy a device, you are getting hundreds if not thousands of work-hours into your lab. 

I come from the microscopy world, and in there we have a great example: [The Open SPIM](https://openspim.org/Welcome_to_the_OpenSPIM_Wiki) . As you can see, making open hardware takes a very big effort, and in any case you always rely on products manufactured industrially, such as a camera or a laser. 

**TL;DR**:
In short, I think your wording can be improved either way: you commit to the cause of preventing researchers from patenting and companies should never be built around scientific research, or you acknowledge that companies are necessary and the problem is rooted on researchers' attitude.

**Comment by aquilesC** [comment]
Thanks for the thorough answer to my thoughts. 

I also agree on the fact that patents are a thing from the past, but it is a reality that that when you seek out investments, a patent is the only thing that generates any kind of guarantee at the eyes of the investor.   

> 1 - As far as I know, in order to patent something, the person/group/university requesting the patent has to keep all the work related to the patent private until the patent is granted/approved

This is not exactly true. You have two instances, the patent application and the granting. The embargo works only until the applications is published (and that is faster than the review process of a paper). 

Moreover, patents are much more thorough than papers in the sense of what they ask from you. This are the The comments of the [European IPR](https://iprhelpdesk.eu/):

> Patenting entails the grant exclusive right to prevent third parties from commercially exploiting - making, using, offering for sale, selling or importing the invention, which is protected by patent for a limited period of time (generally 20 years). In return for this monopoly, the patent owner is required to disclose the technical  information  on  the  invention  in  order  for  others  to  access  it  and 
continue to innovate based on it.  

You can find more information in [the factsheet](https://iprhelpdesk.eu/sites/default/files/newsdocuments/Patenting_v._publishing_0.pdf). 

> 2 - The cost for a patent can be quite high. This can bring a bias on who can patent something.

That is true, but is it a concern? I mean, if you are a researcher and your job is to do '*science*', your objective shouldn't be patenting things, right? So, high costs for patents are not preventing you from publishing, etc. If you worry about using patents as a measure for local innovation, then the discussion is different, but not from the perspective of a researcher. 

> 3 - yes, it is true that for research there is a "waiver" for the patent and researchers can replicate patented tools. BUT scientific equipment is NOT ONLY used inside research institutions. Schools, hospitals, diagnostic facilities are a few examples. Are they also allowed to simply copy patented tools and use their clones for their needs?

Yes, they are allowed to and they can build on. You can check the paragraph I quoted above. The patent only applies for commercial applications. You can't build and sell a device that uses my technology without reaching an agreement with me. You can't sell a service based on a patented technique, etc. Devices for diagnostics are a different topic because they require certifications; who is able to pay for those certifications? But yes, if you have no business associated with the diagnosis, then you can use whatever you want. 

I've checked some of the companies that you listed. For example, **OpenEphys** is built around [Intan](http://www.intantech.com). Intan is a for-profit, *fully closed* company. OpenEphys is just adding to their value chain. 

OpenQCM is very misterious. I could only find the software, but not any single spec on hardware. They are also backed by a non-open company: Novatech (spinoff of INAF), so I assume they operate on a similar fashion to OpenEphys, increasing the value chain of a parent company.

**SanWorks** is different, because they go one step further in their open hardware approach. However, check [this note](https://sites.google.com/site/pulsepalwiki/assembling-pulse-pal/pulse-pal-bill-of-materials) where they warn that they core electronic component, even if open, can't be bought anymore. Anybody *could* build it, but you need a chinese factory to do it in order to compensate the immense amount of work. 

Together with SanWorks, OpenBCI and OpenPCR, are built on Arduino, which makes me think that the innovation is more focused on design than on real hardware (not that this makes it less valid), but also points to the fact that the true innovator in all this is Arduino, and that is the real company people should look at. 

The other companies were more confusing, so I couldn't really understand what/how they do things. 
 
> But it seems their business model (forgetting about patents) works

This may be true, but the question is, do you understand their business model? Some are non-profits, some receive funding from larger companies, some leverage on work done in research institutes publicly funded, and since there are *no patents*, there are also no licenses. Is it really fair that few make money at the expense of the people who payed taxes? Making it open doesn't mean there's no money involved. I think it is also worth reflecting on this. 

> Prusa Research, Arduino, Adafruit and Sparkfun

Those are the real innovators, in my opinion, regarding the open-hardware approach. Still, however, they work around hardware which is not open (such as the Atmel chips). Now, can you compare those companies to companies producing devices that just in materials may be spending between 50 and 100k€, and in markets that are small by definition? As soon as you stop getting *free-money* like in academia, would you be willing to open up all your knowledge to any other to scoop you? It would be the equivalent of working acquiring data for a year, and before even starting to analyze it or writing a paper you say: *hey, I have this dataset, feel free to do whatever you want*. Are researchers willing to go *this open*? 

> Companies have to be paid for this effort, what they should not have is a ludicrous profit margin on every tool they sell. I don't have hard data on companies operating profits, sorry, but for example ~400 bucks for a plastic holder smells like a rigged game. 

This is not fair at all. It would be equivalent to saying 1000€ for a phone seems a rigged game. The business model of Eppendorf is not yours to judge. You can design and print your own holder if you want, perhaps you are even free to sell them. I thought the discussion was regarding tools born in academia and transferred out and the responsibility researchers have to make that research available to anyone. I do agree that when you do research in a field where there are big industries involved, such as  pharma, prices of equipment are skewed because you have *very wealthy* and *poor* customers in the same exact market. Your are completely free not to buy things from Eppendorf, so why do you do it? Perhaps because their quality (at least the perceived one) is superior to the one of the Chinese knockouts? 

As a side note, Eppendorf revenue was €651.5 million, while its Operating Income was €140.1 million ([Wikipedia](https://en.wikipedia.org/wiki/Eppendorf_(company))).

> is that companies need to change their operating methods, from leveraging scarcity using patents to leverage customer service by using open source paradigms.

This, on the one hand is dictated by the market. If companies using an open approach become more successful than competitors, it will naturally happen. For instance, this can be helped if researchers opt for buying open hardware even if it is more expensive than closed hardware. Would you be able to justify the choice with your funding agency? 

Secondly, this can also be changed by changing the mentality of researchers who spin-off companies. Now, I wonder how many spin-offs would be born if your premise is to build something anybody can copy and sell, including competitors. As soon as you validate the tech, market fit, etc. anybody can scoop you out of business. As soon as you certify your technique for diagnosis (with all the costs associated), anybody else can sell much cheaper just because it doesn't have to offset all those costs.  This is a personal experience as well, two proposals for funds for tech transfer, differing only on the open nature of the project had two opposite outcomes. 

Thirdly, comparing hardware to software is not fair. Open source software as a commercial product works around service agreements. And this works because the investment is very small. If I give a team of developers 100k€ or 200k€ to build a product, I could build a clone of Github, which as you know is valued in the billions of dollars. If you give me 200k€, I can hardly build a hardware prototype and iterate over it. Do you think you can develop the ATmega chips of the arduinos with just 100k€? [Here you have a nice article regarding the price to build software products](https://thenextweb.com/dd/2013/12/02/much-cost-build-worlds-hottest-startups/). 

The business model of the Arduino works because they leverage the low price of a product (the micro-controller) that was not developed by them. The biggest part of the risk was transferred up the chain, and now you have a nice open-hardware tool. On the other hand, Arduino is generating value to Atmel, because it doesn't matter if it's the official board or a copy, they all use the same, micro-controller. 

So, I know I'm playing devil's lawyer here. I am in the in between situation of being a former academic researcher advocating for openness and transitioning into a spinoff. Therefore, I have to be realistic around a lot of concerns, and one is that by making our products open, we wouldn't get nor funding nor investment. So you wonder, what is better? Not doing anything or at least knowing that you can bring a technology to market, making it available to anyone who is willing to buy? In the process, money will reach the university, effectively increasing the opportunities in academia. On the other hand I do believe companies can contribute to open-source software, for example, without hindering their business models. 

What I think is a good path, is to make it clear to researchers how to open up their technology, even if it will never arrive to market as a product. I also think, what you mention is a purely academical problem, not a problem of companies. And this is enough of a titanic task; just look at the methods section of the papers; researchers on average don't like explaining what they do and how they do it. Are they afraid of being copied? There could be thousands or millions of blog articles explaining how to do stuff, how to build a setup, etc. Who pays for those hours? Are funding agencies wiling to judge those contributions to the community equally relevant as a paper? 

Patenting your invention is not, in my opinion, the root cause of the closed nature in the academic environment. I do see an opportunity in opening hardware and software, because it opens up the possibilities of increasing the value of the product thanks to the creativity of others. It also opens the access to researchers in less wealthy nations, etc. But innovation is not free and you need to account for that. If you are a researcher, you can focus on how to improve your environment (if you find collective support, which is a big assumption in this context). It is not up to you to judge business models of companies such as Eppendorf, but you are free (are you?) to buy from others who do support your view, that is your real (even if limited) power.

**Comment by amchagas** [comment]
Hi, sorry for the delay in replying! and thanks for taking the time as well :)

> I also agree on the fact that patents are a thing from the past, but it is a reality that that when you seek out investments, a patent is the only thing that generates any kind of guarantee at the eyes of the investor.

I don't agree with this, I definitely heard it many times, and I believe it is the way most investors work. At the same time, I think this is changing. Here is an example of a hardware company that is Open Source and received venture capital https://sosv.com/portfolio/opentrons/

> This is not exactly true. You have two instances, the patent application and the granting. The embargo works only until the applications is published (and that is faster than the review process of a paper).

So in any case, a publication is even more delayed when a patent comes to the mix. And another question I have about this is: Are all patent granting agencies taking the same time to analyse things and make a decision?

> Moreover, patents are much more thorough than papers in the sense of what they ask from you. 

Agree that we need to drastically improve the methods section of papers. Maybe even use the level of detail required for patenting, but the description has to be on the paper, and not on a separate document that is stored by a different entity. Moreover, not all papers have material that can/will be patented.


>    2 - The cost for a patent can be quite high. This can bring a bias on who can patent something.
That is true, but is it a concern? I mean, if you are a researcher and your job is to do 'science', your objective shouldn't be patenting things, right? So, high costs for patents are not preventing you from publishing, etc. If you worry about using patents as a measure for local innovation, then the discussion is different, but not from the perspective of a researcher.

In principle, yes, researchers are researching. But they are not doing that in a vacuum right? I mean, spending money and time to patent things, takes money away from other things, as their institutions do not have infinite pools of cash and employees. Or in the example you gave, spending money on patents, could as well be taking money away from publishing. Also, I don't like this thought that researchers are only supposed to be researching, in my view is what can/could lead to the public knowing very little about what we are doing. which can lead to mistrust in science and what is worst in bogus science gaining the spot light. I know this is not the main part of our discussion here, but I think it falls in the same logic. 

> Yes, they are allowed to and they can build on. You can check the paragraph I quoted above. The patent only applies for commercial applications. You can't build and sell a device that uses my technology without reaching an agreement with me. You can't sell a service based on a patented technique, etc. Devices for diagnostics are a different topic because they require certifications; who is able to pay for those certifications? 

Thanks for the clarification!

>But yes, if you have no business associated with the diagnosis, then you can use whatever you want.

do you happen to know anyone/institution who is doing that systematically? It would be interesting to know about real cases. 

About Sanworks, and the microcontroller they were using, It was a Maple board. Main difference from the Arduino was that this one had much more processing power. Although it got discontinued, many other compatible (and even more powerful) boards emerged. ESP32, ESP8266 are examples  They are also open source devices.

> I've checked some of the companies that you listed. For example, OpenEphys is built around Intan. Intan is a for-profit, fully closed company. OpenEphys is just adding to their value chain.

This brings me to the point of CPUs (and other ICs) not being open and what it means to open source hardware. In short: my definition is that if I can buy all the building blocks of a piece of hardware using freely shared blueprints, and the software to run it, then it is Open Source Hardware. Using the example of computer processors, does it mean that software that is written using computers that have those processors are also not open source? (this is more of a provocation :P ) On the same note, there are open source projects developing processors, or at least the standards to produce them https://riscv.org/


OpenQCM : completely surprised. this was not the case some months ago. Going to get in touch with them.

> Together with SanWorks, OpenBCI and OpenPCR, are built on Arduino, which makes me think that the innovation is more focused on design than on real hardware (not that this makes it less valid), but also points to the fact that the true innovator in all this is Arduino, and that is the real company people should look at.

Completely disagree. so GNU/Linux is the real thing we should look at and all the things that came after/derived from it are not real innovators? Or say you wrote code using google's Tensor Flow library. Are you not innovating?

> Some are non-profits, some receive funding from larger companies, some leverage on work done in research institutes publicly funded, and since there are no patents, there are also no licenses. 

About not having patents and not having licenses, this is not true. There are many licenses for open source projects. For Hardware for instance, CERN has a license. The Open Source Hardware Association has a license. Creative Commons, GNU GPL, and many others. and they also manage to enforce them when they are infringed https://www.gnu.org/philosophy/enforcing-gpl.en.html

> This is not fair at all. It would be equivalent to saying 1000€ for a phone seems a rigged game. The business model of Eppendorf is not yours to judge.

Agreed that 1000€ for a phone seems like a rigged game. I would say judge is a very strong word, but at the same time, we cannot forget (and I'm sorry for what is going to look like a jump here, but this would be very long to explain in text) that we live in a system that is only able to produce a certain amount of wealth per unit of time, and for someone (or some company) to make a shit ton of money, someone else has to be left without. You cite Eppendorf operating profits. We can't forget that we are talking about profits here. They paid all their costs, all their employees, all their research and still have millions to cash in the bank. Don't get me wrong. I think if people do hard work they deserve to be paid for their efforts. And I also agree that there will be differences in rewards and payments, depending on what you do, how many years you had the opportunity to study, etc.  What I don't agree with are the extreme inequalities we see in the world today. This doesn't happen out of thin air. it gets done by humans, that are making decisions everyday. The current idea that companies are machines that maximize profit is ludicrous. I don't think we will get very far this way.  

> This, on the one hand is dictated by the market. 

The market as "an invisible" institution annoys me. people are the market. Companies are the market. governments are the market.

About spin off companies and how many researchers would operate on those conditions, your Arduino example is good here. How many cheap arduino clones are there? coming much cheaper from China? Arduino still is, as far as I know, a profitable company. They protect their brand name and logos, instead of the product. They offer good boards, an entire community where people co-develop and resolve issues. In other words, they generate value with good service and community infrastructure. 

Prusa Research and Ultmaker (at least until their UM2 models) are open source companies. They are immensely cloned (a prusa printer costs about 950 bucks and a clone 200). They too have a fantastic customer service (we have one of each in the lab), and protect their brands. As a consumer, I buy from them because I know that whenever there are problems, I will get solutions fast From what it is know, both companies are doing very well and keep bringing innovation after innovation to the 3D printing market. And as far as know Prusa didn't have a gigantic initial investment (as you mention that 100-200k won't get you very far when building hardware). Maybe they also have a extreme profit margin, which is not cool. but in this case, if I'm unhappy with that, I can replicate the system myself..

They are indeed using processors that are not open. And here is another moment to make a point. I'm not saying that overnight we are going to replace the business models existing out there with a single magic bullet. There is no such thing. But I believe that for research and education (mostly funded with public money, generate the research and prototypes of the tools that will be used in the same industry) companies should start acting more as service providers. CERN has been operating that way for years with their tools/instruments providers. I understand that CERN might be a special case, since they have a lot of pull to negotiate.

> For instance, this can be helped if researchers opt for buying open hardware even if it is more expensive than closed hardware. Would you be able to justify the choice with your funding agency?

reasons for using open instead of closed:
Faster to customize, cheaper to repair, not locked in to a vendor, better reproducibility (more details here: http://bit.ly/2B1Htg3). Moreover, in real life, I haven't seen a single case where open was more expensive then closed. If you know of one, please share

> Now, I wonder how many spin-offs would be born if your premise is to build something anybody can copy and sell, including competitors. As soon as you validate the tech, market fit, etc. anybody can scoop you out of business. As soon as you certify your technique for diagnosis (with all the costs associated), anybody else can sell much cheaper just because it doesn't have to offset all those costs. 

This is only partially true, as certification for diagnostic devices are given to the companies producing them. Normally diagnostic techniques are created and validated inside Academia. The creation of the commercial device, in a company, that them retains certification for the device. So even if the devices were open, the company cloning them would still have costs to get the certification, I agree that these costs are not the same as the company that created the device in the first place, but then again the first company to market was still the first company to market. 

Patents by themselves are not the main issue, but rather how they are used (and even abused) sometimes. I mentioned 3D printing already, but we are only seeing this boom in 3D printers, drop in price, big adoption, after the patents for the FDM systems expired in 2009. https://en.wikipedia.org/wiki/3D_printing#History since 1988 we have the technology. Very very little adoption/use before the patents expired (as compared to today).

I hope your this new phase you are going into develops well! I'm sorry to hear that when you tried to raise money for an open project, things didn't go so well. Maybe (if you haven't yet) you should try to go after funds, trusts, agencies that already have a track history with funding open projects? Mozilla, Sloan, shuttleworth, Volkswagen foundations might be good places to start? Also, (again maybe you already have read) there are three books I like a lot on open things and have a perspective from people that are on the ground (and probably make better points than me): Open Source everything Manifesto (for the philosophical point of view of opening everything), The Cathedral and the Bazaar (for open source software) and Building Open Source Hardware (for open source hardware) 

 About our text issue, of changing the wording, why don't you fork the project here and make the changes you'd like to see? this way we can more practically come to a solution! (don't get me wrong, I like the conversation here, and don't mind that we keep going, but it would be nice to see your contributions directly on the project, and I think I'm may be too biased to make significant changes here)

**Comment by aquilesC** [comment]
Thanks for your comments! 
I think it is a topic that requires a lot of thought. I needed to engage in this discussion to try to understand what can be done, and how people pushing for open-work think. Sorry for playing de devil's lawyer, I do agree with many of the points you make, but I also have concerns on how realistic it is to achieve changes in the short term, and wanted to understand your position.

> there are three books I like a lot on open things and have a perspective from people that are on the ground (and probably make better points than me):

I will check them. Surprisingly, the only one which is actually an open book is *The Cathedral and the Bazaar* (even though printing rights are exclusive to O'Reilly). 

> why don't you fork the project here and make the changes you'd like to see?

I will think about it. With code, spelling mistakes, etc. where there is a sort of *non-objectionable* way, I would do it. For wording of a project, that seems more personal for the people who are actively engaged and not passers-by like me. Anyway, if I do come up with something you'll see the request arriving.


I will close this issue here, as I think the discussion extended for too long.
